lib/process-services/src/lib/form/widgets/people/people.widget.ts

```diff
@@ -135,7 +137,7 @@ export class PeopleWidgetComponent extends WidgetComponent implements OnInit {
 
     getDisplayName(model: LightUserRepresentation) {
         if (model) {
-            const displayName = `${model.firstName || ''} ${model.lastName || ''}`;
+            const displayName = `${model.firstName || ''} ${model.lastName || ''} <${model.email}>`;
```

Comment:
there are checks for firstName and lastName to be null with empty string fallbacks, but nothing for the email... so you will render `<>` in case email is missing

Reply:
@DenysVuika plz check PR once again, it was wrong file